src/main/java/com/ramcel/cinema/reservation/functionalities/screening/ScreeningService.java

```diff
@@ -0,0 +1,11 @@
+package com.ramcel.cinema.reservation.functionalities.screening;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public interface ScreeningService {
+
+    public List<Screening> findScreenings(LocalDateTime dateTime);
```

Comment:
Czy użycie `public` w definicji abstrakcyjnej metody w interface jest potrzebne?

Reply:
Efektywnie nic nie daje, ale lubię jak to jest jasno wpisane i nie muszę potem się pięć razy zastanawiać, jak mam już zlasowany mózg po kiku godzinach pisania.